lib/utils/aws/aws.go

```diff
@@ -68,6 +68,9 @@ const (
 	AmzJSON1_0 = "application/x-amz-json-1.0"
 	// AmzJSON1_1 is an AWS Content-Type header that indicates the media type is JSON.
 	AmzJSON1_1 = "application/x-amz-json-1.1"
+
+	// MaxRoleSessionName is the maximum length of the role session name used by the AssumeRole call.
+	MaxRoleSessionName = 64
```

Comment:
nit: leave a link reference
```suggestion
	// MaxRoleSessionName is the maximum length of the role session name used by the AssumeRole call.
	// https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-quotas.html
     	MaxRoleSessionName = 64
```